### PR TITLE
Daily jenkinsfile update

### PR DESCRIPTION
**Pull request by ingwinlu**
# Purpose

* lowerCamelCaseEverywhere
* add docker registry cleanup script

# Checklist

- [x] commit messages are fine ("module: short statement" syntax and refer to issues)
- [x] affected documentation is fixed
- [x] release notes are updated (doc/news/_preparation_next_release.md)

@markus2330 please review my pull request

### CODE REVIEW

**Comment by ingwinlu** [comment]
jenkins build daily please

**Comment by ingwinlu** [comment]
jenkins build daily please

**Comment by ingwinlu** [comment]
jenkins build daily please

**Comment by ingwinlu** [comment]
jenkins build daily please

**Comment by ingwinlu** [comment]
actually the garbage collect run is restarting itself because the service is set to restart automatically...

**Comment by ingwinlu** [comment]
jenkins build daily please

File: scripts/jenkins/Jenkinsfile.daily [hunk 1]
```diff
@@ -80,13 +84,42 @@ def cleanup_docker_images(target_node) {
     }]
 }
 
-def generate_file_cleanup_stages() {
+def generateDockerRepoMaintanenceStages() {
+    def nodes = nodesByLabel label:"docker&&registry"
+    def tasks = [:]
+
+    nodes.each { target_node ->
```
Comment (markus2330) [inline scripts/jenkins/Jenkinsfile.daily]: Why target_node not camel case?

File: scripts/jenkins/Jenkinsfile.daily [hunk 2]
```diff
@@ -80,13 +84,42 @@ def cleanup_docker_images(target_node) {
     }]
 }
 
-def generate_file_cleanup_stages() {
+def generateDockerRepoMaintanenceStages() {
```
Comment (markus2330) [inline scripts/jenkins/Jenkinsfile.daily]: please some docu

File: scripts/jenkins/Jenkinsfile.daily [hunk 3]
```diff
@@ -80,13 +84,42 @@ def cleanup_docker_images(target_node) {
...
+    def nodes = nodesByLabel label:"docker&&registry"
+    def tasks = [:]
+
+    nodes.each { target_node ->
+        tasks << cleanupDockerRegistry(target_node)
+    }
+    return tasks
+}
+
+def cleanupDockerRegistry(target_node) {
```
Comment (markus2330) [inline scripts/jenkins/Jenkinsfile.daily]: please some docu

File: scripts/jenkins/Jenkinsfile.daily [hunk 4]
```diff
@@ -80,13 +84,42 @@ def cleanup_docker_images(target_node) {
...
+    def taskname = "cleanupDockerRegistry/${target_node}"
+    return [(taskname): {
+        stage(taskname) {
+            node(target_node) {
+                withCredentials([usernameColonPassword(credentialsId:
+                'docker-hub-elektra-jenkins', variable: 'AUTH')]) {
+                    docker.image('anoxis/registry-cli')
```
Comment (markus2330) [inline scripts/jenkins/Jenkinsfile.daily]: What is anoxis?

File: scripts/jenkins/Jenkinsfile.daily (hunk 1 above)
Comment (markus2330) [reply scripts/jenkins/Jenkinsfile.daily]: It does not really match with your description of the PR: lowerCamelCaseEverywhere

And I suggest to actually do that, mixing CamelCase and other_ways is not a good idea. When someone else starts modifying the code, inconsistencies will appear.

**Comment by ingwinlu** [comment]
jenkins build daily please

**Comment by markus2330** [comment]
I just noticed that "jenkins build daily please" is not described in doc/GIT.md

Please keep docu up-to-date and consistent, otherwise it is very difficult for others.

**Comment by ingwinlu** [comment]
yeah because I only fixed it yesterday (it did not work before)

**Comment by ingwinlu** [comment]
jenkins build daily please

**Comment by markus2330** [comment]
Thank you, great job!

**Pull request merged**